arm-hdinsight/2015-03-01-preview/swagger/cluster.json

```diff
@@ -404,6 +404,12 @@
                 },
                 "configurations": {
                     "type": "object",
+                    "additionalProperties": {

```

Comment:
This is breaking changes because instead of any kind of object it's became a dictionary of dictionaries of strings.

C#:
```csharp
Dictionary<string, Dictionary<string, string>>
```

TypeScript:
```ts
{ [name: string]: { [name: string]: string } }
```